src/App.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Quotes from './components/quote';
+import NavBar from './components/navbar';
+import Home from './components/home';
 
-class RenderCalc extends React.Component {
+class RenderPage extends React.PureComponent {
   constructor(props) {
     super(props);
     this.state = {};
   }
 
   render() {
     return (
-      <Calculator />
+      <>
+        <Router>
+          <NavBar />
+          <Routes>
+            <Route path="/calculator" element={<Calculator />} />
+            <Route path="/" element={<Home />} />
+            <Route path="/quotes" element={<Quotes />} />
+          </Routes>
+        </Router>
+      </>
     );
   }
 }
-export default RenderCalc;
+export default RenderPage;
```

Comment:
### OPTIONAL

- [ ] Kindly make your App.js component function-based, this will enable you write less code, and function-based components are easier to understand 👍🏿

Reply:
Thank you!